Python/compile.c

```diff
@@ -854,20 +854,26 @@ compiler_set_qualname(struct compiler *c)
 /* Allocate a new block and return a pointer to it.
    Returns NULL on error.
 */
+static basicblock *
+new_basicblock()
+{
+    basicblock *b = (basicblock *)PyObject_Calloc(1, sizeof(basicblock));
+    if (b == NULL) {
+        PyErr_NoMemory();
```

Comment:
I always forget this myself, but `PyErr_NoMemory` returns `NULL`, so you can replace
```C
    PyErr_NoMemory();
    return NULL;
```
with
```C
    return PyErr_NoMemory();
```

Reply:
But it would require a cast as `PyErr_NoMemory` return `PyObject *` whereas this function returns `basicblock *`

Reply:
Good point, probably best as it is then.